decksite/data/deck.py

```diff
@@ -502,6 +502,15 @@ def load_conflicted_decks() -> List[Deck]:
         )"""
     return load_decks(where, order_by='d.decklist_hash')
 
+def load_confidence(decks: List[Deck]) -> None:
+    # Confidence scores for decks not in the queue get deleted by the hourly maintenance job, so just get all
+    sql = 'SELECT deck_id, score FROM confidence'
+    conf = {}
+    for row in (Container(r) for r in db().select(sql)):
+        conf[row.deck_id] = row.score
+    for deck in decks:
+        deck.confidence = f'{conf[deck.id]}%' if deck.id in conf else ''
```

Comment:
This is our first (?) use of more than a simple variable in `{}` inside a string. I'm not sure how I feel about it. f''-strings are super useful but rather like mustache templates I'd hate to see them getting complicated. Probably not a big deal.

Reply:
I thought we had some of these already, but I'll replace it with a `.format` for now and we can worry about where to draw the line another time :)